make_dashboards/templates/portal/index.html

```diff
@@ -117,13 +114,9 @@
             }
         });
         
-        // Header clock
-        updateClock();
-        
-        // Cron job runs on the hour, every hour. Get the web page to reload at 5 past the next hour
-        var reloadDelay = moment().add(1, 'hours').startOf('hour').add(5, 'minutes').diff();
-        setTimeout(function(){ location.reload(); }, reloadDelay );
```

Comment:
Yeah - this was leftover code from when I copied the template from the dashboard. The TV screens need to reload themselves to stay up to date, but this is the portal template which people load in a web browser.